### PR TITLE
Ported Cloud version of the Manage Workspaces doc to Enterprise

### PR DESCRIPTION
**Pull request by jwitz**
Part 4 of this Issue: https://github.com/astronomer/docs/issues/179

While the Managing Workspaces doc has a long ways to go, this PR brings some of the vital fixes from the Cloud doc over to Enterprise. I also updated some references to Astronomer terms per our style guide across both Cloud and Enterprise versions.

The only new content here is a rewording of the last sentence in the Enterprise "Workspace Permissions" section.

### CODE REVIEW

File: enterprise/next/04_deploy/01_manage-workspaces.md [hunk 1]
```diff
@@ -69,12 +66,14 @@ The former will link you directly to your DAG Dashboard on Airflow itself. Your
...
 
-**Each deployment will run in a separate Kubernetes namespace, so they'll all get resources and maintain metadata independently. You should assume that each deployment is unaware of the others.**
+> **Note:** All Airflow Deployments run in an isolated Kubernetes namespace, which means resources will be provisioned independently and data will be kept isolated from the rest. You can assume that each Airflow Deployment is unaware of the others, even within the same Workspace.
 
 ## User Management
 
-If you navigate over to the "Users" tab of your Workspace Dashboard, you'll be able to see who has access to the Workspace. If you'd like to share access to other members of your organization, invite them to a workspace you're a part of. Once members, they'll have access to _all_ Airflow deployments under that workspace.
+If you navigate over to the "Users" tab of your Workspace Dashboard, you'll be able to see who has access to the Workspace.
+
+If you'd like to share access to other members of your organization, invite them to a Workspace you're a part of. Once members, they'll have access to _all_ Airflow Deployments under that Workspace with varying permissions.
```
Comment (paolaperaza) [inline enterprise/next/04_deploy/01_manage-workspaces.md:75]: @jwitz Can you update the language in this line and below to reflect Deployment-level RBAC? We have _some_ language in the Quickstart:

> Once you're a member of a Workspace, Deployment Admins can grant you varying levels of access to the Airflow Deployment(s) within that Workspace.

Deployment-level RBAC is applicable for Cloud and Enterprise 0.23, so we can keep a distinct version for v0.16 that doesn't have that reference.

**Review by paolaperaza — CHANGES_REQUESTED** [review]
@jwitz This looks good! Easy enough changes. I added a comment, but can you make sure to tweak language at the bottom of this doc (across versions + Cloud/Enterprise) to reflect Deployment-level RBAC and link out to the right spots?

Looks good otherwise - I'll take a look again from there. Thank ya!

**Comment by jwitz** [comment]
Sounds good @paolaperaza , I updated these sections in `cloud` and `next`, and I linked out to our docs on the feature where possible.

File: enterprise/next/04_deploy/01_manage-workspaces.md (hunk 1 above)
Comment (jwitz) [reply enterprise/next/04_deploy/01_manage-workspaces.md]: Having read through the changelog, I further updated this to reflect that Workspace user roles are similar to Deployment user roles. This means that we don't have to have two separate sections on user management that are mostly redundant.

File: enterprise/next/04_deploy/01_manage-workspaces.md [hunk 2]
```diff
@@ -4,59 +4,56 @@ navTitle: "Manage Workspaces"
...
 
-Here, you have a high-level overview of all of the active Airflow deployments you have running in that given workspace. In this case, we only have one cluster spun up.
+Here, you have a high-level overview of all of the active Airflow Deployments you have running in that given Workspace. In this case, we only have one cluster spun up.
 
 From this screen, you can:
 
-1. Create new Airflow deployments
-2. Manage user access to the workspace
+1. Create new Airflow Deployments
+2. Manage user access to the Workspace
```
Comment (paolaperaza) [inline enterprise/next/04_deploy/01_manage-workspaces.md:38]: Remove period on step 3 here?

File: cloud/stable/03_deploy/01_manage-workspaces.md [hunk 3]
```diff
@@ -18,42 +18,42 @@ Once logged in, you'll land on a view that will direct you to create a new **Wor
...
-Here, you have a high-level overview of all of the active Airflow deployments you have running in that given workspace. In this case, we only have one cluster spun up.
+Here, you have a high-level overview of all of the active Airflow Deployments you have running in that given Workspace. In this case, we only have one cluster spun up.
 
 From this screen, you can:
 
-1. Create new Airflow deployments
-2. Manage user access to the workspace
+1. Create new Airflow Deployments
+2. Manage user access to the Workspace
 3. Generate tokens for CI/CD systems via service accounts.
```
Comment (paolaperaza) [inline cloud/stable/03_deploy/01_manage-workspaces.md:39]: Remove period from step 3?

File: enterprise/next/04_deploy/01_manage-workspaces.md [hunk 4]
```diff
@@ -69,12 +66,14 @@ The former will link you directly to your DAG Dashboard on Airflow itself. Your
...
 
 ## User Management
 
-If you navigate over to the "Users" tab of your Workspace Dashboard, you'll be able to see who has access to the Workspace. If you'd like to share access to other members of your organization, invite them to a workspace you're a part of. Once members, they'll have access to _all_ Airflow deployments under that workspace.
+If you go to the **Users** tab of your Workspace Dashboard, you'll see who has access to the Workspace.
+
+If you'd like to share access to other members of your organization, invite them to a Workspace you're a part of. Once your team members are part of your Workspace, Deployment Admins can grant them varying levels of access to Airflow Deployments within the Workspace. Likewise, Workspace Admins can grant them varying levels of access to the entire Workspace.
 
-## Workspace Permissions
+An exact breakdown of user roles and their respective levels of access can be found in [Manage User Permissions on Astronomer Cloud](/docs/cloud/stable/manage-astronomer/workspace-permissions/).
```
Comment (paolaperaza) [inline enterprise/next/04_deploy/01_manage-workspaces.md:77]:
```suggestion
An exact breakdown of user roles and their respective levels of access can be found in [Manage User Permissions on an Astronomer Workspace](/docs/enterprise/stable/manage-astronomer/workspace-permissions/).
```

**Review by paolaperaza — APPROVED** [review]
Found one typo re: Enterprise doc link, but otherwise looks good - @jwitz. Will you apply these changes to 0.16, EXCEPT for references to deployment-level rbac?

**Pull request merged**